docs/technical-reference/build-test-run.md

```diff
@@ -147,7 +147,7 @@ Other distributions are likely to offer Maven in their official package reposito
 ### Node.js and npm {#nodejs}
 
 
-The OpenRefine webapp requires [Node.js](https://nodejs.org/en/download/) and npm to install package dependencies. We require Node.js 16 or newer.
+The OpenRefine webapp requires [Node.js](https://nodejs.org/en/download/) and npm to install package dependencies. We require Node.js 20 or newer.
```

Comment:
```suggestion
The OpenRefine webapp requires [Node.js](https://nodejs.org/en/download/) and npm to install package dependencies.
```

Consider dropping version reference or replacing it with something generic like "We generally require the latest LTS version of Node.js."

Reply:
I'd rather keep it here and drop it from the Cypress instructions because someone trying to get Cypress to run has probably already met the dependencies to compile and run OpenRefine anyway.

Reply:
OK